src/App.js

```diff
@@ -1,13 +1,24 @@
-import Navbar from "./components/Navbar/Navbar";
-import Sidebar from "./components/Sidebar/Sidebar";
+import { Route, Routes } from "react-router-dom";
+import {
+	Archive,
+	Login,
+	Navbar,
+	NotesPage,
+	Signup,
+	Trash,
+} from "./components/export";
 
 function App() {
 	return (
 		<>
 			<Navbar />
-			<div className="main">
-				<Sidebar />
-			</div>
+			<Routes>
```

Comment:
`Sidebar` comp also should be here, if you're using grids. Placing it on every page is not optimal solution. Cause `sidebar` isn't changing, while the page is!

Reply:
yeah i know , but i don't want them in my signup/login page thats is why had to import it to every comp which needs it.